Xamarin.Essentials/Culture/Culture.shared.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Globalization;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Culture
+    {
+        public static string InstalledUICulture =>
+            PlatformInstalledUICulture;
+
+        public static CultureInfo GetCurrentUICulture() =>
+            PlatformGetCurrentUICulture(null);
+
+        public static CultureInfo GetCurrentUICulture(Func<string, CultureInfo> mappingOverride) =>
+            PlatformGetCurrentUICulture(mappingOverride);
+
+        public static void SetCurrentUICulture(CultureInfo cultureInfo) =>
```

Comment:
Just a thought here, but the "UI" does seem to indicate that this is just UI, but we are setting the thread culture as well. I am not sure if this may cause confusion for developers?